consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/PostMergeContext.java

```diff
@@ -99,7 +99,9 @@ public void setIsPostMerge(final Difficulty totalDifficulty) {
 
     if (oldState.isEmpty() || oldState.get() != newState) {
       newMergeStateCallbackSubscribers.forEach(
-          newMergeStateCallback -> newMergeStateCallback.onNewIsPostMergeState(newState));
+          newMergeStateCallback ->
+              newMergeStateCallback.onCrossingMergeBoundary(
```

Comment:
this is a clarifying name change at transition, but post-merge this event fires when the client starts up even if we are long past TTD.  I agree NewIsPostMergeState is not a great name, but CrossingMergeBoundary is a bit misleading also considering the startup behavior.